crates/oxc_mangler/src/lib.rs

```diff
@@ -305,7 +305,8 @@ impl<'t> Mangler<'t> {
         let mut slots = Vec::from_iter_in(iter::repeat_n(0, scoping.symbols_len()), temp_allocator);
 
         // Stores the lived scope ids for each slot. Keyed by slot number.
-        let mut slot_liveness: std::vec::Vec<FixedBitSet> = vec![];
+        // Using std::vec::Vec because FixedBitSet implements Drop
+        let mut slot_liveness: std::vec::Vec<FixedBitSet> = std::vec::Vec::with_capacity(64);
```

Comment:
Not sure what the heuristic is which makes 64 a reasonable guess at required capacity.